src/main/resources/jsonToMT103/component.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the license header year

Reply:
Changed in commit 96d4dc498094daee16277501378ee52fbb484dd1